### PR TITLE
Add version to use release pipeline on quay

### PR DESCRIPTION
**Pull request by pacospace**
- Correct query
- Introduce version to release on quay

### CODE REVIEW

File: thoth/package_release_job/__init__.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+"""This is Thoth package-release-job."""
+
+from thoth.common import __version__ as __common__version__
+from thoth.common import init_logging
+from thoth.storages import __version__ as __storages__version__
+from thoth.python import __version__ as __python__version__
+
+__version__ = "0.6.0"
+__service_version__ = f"{__version__}+\
```
Comment (fridex) [inline thoth/package_release_job/__init__.py:27]: You can put these values directly to `app.py` no need to create this directory structure.

Comment (pacospace) [reply thoth/package_release_job/__init__.py:27]: Don't we want to run https://github.com/thoth-station/kebechet/tree/master/kebechet/managers/version#kebechet-version-manager here?

Comment (fridex) [reply thoth/package_release_job/__init__.py:27]: Yes, we want. It respects openshift s2i conventions and checks `app.py` for version management.

File: app.py [hunk 2]
```diff
@@ -30,22 +30,25 @@
...
+__version__ = "0.6.0"
+__service_version__ = f"{__version__}+\
+    storages.{__storages__version__}.\
+        common.{__common__version__}.\
+            lab.{__python__version__}"
 
 init_logging()
 
 _LOGGER = logging.getLogger("thoth.package_releases")
+_LOGGER.info(f"Thoth package-release-job v%s", __service_version__)
```
Comment (fridex) [inline app.py:50]: No need for f-string.

**Pull request merged**